app/services/tracked-data.js

```diff
@@ -6,10 +6,10 @@ import { generateRandomCoordinates } from 'esri-classic-vs-glimmer/utils';
 export default class TrackedDataService extends Service {
   @service stats;
 
-  @tracked routes = [];
+  routes = [];
   @tracked visibleRouteIds = [];
-  @tracked isSelectionRunning = false;
```

Comment:
How does the template (button text) update without this?

Reply:
this is just haphazard trying of things, it breaks the ui in the way you expect